jest/verifyComponent.tsx

```diff
@@ -18,11 +18,11 @@ import {render, fireEvent} from '@testing-library/react';
  * })
  */
 export function verifyComponent(
-  Component: React.ComponentType<any>,
+  Component: React.ComponentType<React.PropsWithChildren<any>>,
```

Comment:
I'm curious about this change. It doesn't look like it does anything.

Here's the type for `React.PropsWithChildren<T>`:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ea4d4e50e11788ef0a8094d7de564ad2f01dfedd/types/react/index.d.ts#L799
```tsx
type PropsWithChildren<P = unknown> = P & { children?: ReactNode | undefined };
```

Passing `any` to it will result in:
```tsx
any & { children?: ReactNode | undefined }
```

which will result in `any`.

Was this a find/replace or did this solve something?

Reply:
This was automatic replacement by [types-react-codemod](https://github.com/eps1lon/types-react-codemod`. We can keep it as `any`.